### PR TITLE
D0C-49 Instructions for installing repo

### PR DESCRIPTION
**Pull request by sarangamborkar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,8 +9,27 @@ For a detailed overview of our available API endpoints, please refer to the Welk
...
+- Python 3.8
+- Python 3.9
+
+Assuming that you have Python and virtualenv installed, set up your environment and install the required dependencies 
+like this or you can install the library using pip:
+
+```
+$ git clone https://github.com/welkincloud-io/sample-apps.git
+$ cd sample-apps
+$ virtualenv venv
```
Comment (reykatz) [inline README.md:35]: I had to specify python3 when setting up my virtualenv:
`virtualenv -p python3 venv`

File: README.md [hunk 2]
```diff
@@ -9,8 +9,27 @@ For a detailed overview of our available API endpoints, please refer to the Welk
...
-
+###  Getting Started
+#### Supported Python Versions
+This library supports the following Python implementations:
+
+-  Python 3.6
+- Python 3.7
+- Python 3.8
+- Python 3.9
+
```
Comment (reykatz) [inline README.md:20]: Can we add a header saying this is Mac installation instructions?
`#### How to Install on Mac OS` or similar?

File: README.md [hunk 3]
```diff
@@ -9,8 +9,27 @@ For a detailed overview of our available API endpoints, please refer to the Welk
...
+Assuming that you have Python and virtualenv installed, set up your environment and install the required dependencies 
+like this or you can install the library using pip:
+
+```
+$ git clone https://github.com/welkincloud-io/sample-apps.git
+$ cd sample-apps
+$ virtualenv venv
+...
+$ . venv/bin/activate
+$ python -m pip install -r requirements.txt
```
Comment (reykatz) [inline README.md]: Same here, I had to use pip3 instead of pip:
`pip3 install -r requirements.txt`
(The `python -m` was not needed)

**Review by reykatz — CHANGES_REQUESTED** [review]
This looks good, thanks for adding the instructions! I've left a couple comments with the commands that worked on my setup with python3.

**Comment by sarangamborkar** [comment]
I added Admin portal security setting instruction in same request. can you please take look and suggest me if any changes required

**Review by reykatz — APPROVED** [review]
Looks good to me

**Pull request merged**